integration-tests/volumes/client/scenarios_panic.yml

```diff
@@ -0,0 +1,23 @@
+scenarios:
+  - name: advertiser_url_prefix_path_filter_missing_slash
```

Comment:
I don't think this tests what we think it does.

If the settings are invalid, contile should fail to launch, so any query would fail. 

I believe we test that with `docker-compose.panic.yml`.

Reply:
Hi @jrconlin! 👋🏻 

I wrote these tests before we concluded that the main thread does not panic on Slack meaning that it wasn't clear what behavior we expect from Contile if the settings are invalid with #311. This tests does demonstrate that Contile didn't not panic when the settings are passed in as a file path.

I will update this now.